s2protocol/decoders.py

```diff
@@ -66,7 +66,7 @@ def read_bits(self, bits):
             if self._nextbits == 0:
                 if self.done():
                     raise TruncatedError(self)
-                self._next = ord(self._data[self._used])
+                self._next = self._data[self._used]
```

Comment:
I think it might cause a problem in Python2. Since there aren't `bytes` in Python2, `self._data[self._used]` is still Python2 `str`. So I decide to write compatibility functions that help to solve this problem.

```
from six import PY3, binary_type

__all__ = 'byte_to_int',


def byte_to_int(x):
    if PY3 and isinstance(x, binary_type):
        return x
    else:
        return ord(x)
```

Sadly I found this PR after I made my own version of Python3 support version 😭.

Reply:
I also opened my version of PR. https://github.com/Blizzard/s2protocol/pull/86